apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/view.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+// disabled CSS class rule due to existing code already
+// that users the non-conformant classnames
```

Comment:
Can these just be updated in follow up PR? :-)

Reply:
My concern is if we change the classnames then posts already published will not style properly. We could update but would need to include the old classnames in the CSS, so my thought was to leave to avoid the extra definitions, but a follow up PR is a good idea to discuss and address.